integration_tests/e2e/reviewPlan/review/reviewPlan.cy.ts

```diff
@@ -2,7 +2,7 @@ import { startOfToday, sub } from 'date-fns'
 import Page from '../../../pages/page'
 import WhoCompletedReviewPage from '../../../pages/reviewPlan/WhoCompletedReviewPage'
 import AuthorisationErrorPage from '../../../pages/authorisationError'
-import ReviewPlanCompletedByValue from '../../../../server/enums/reviewPlanCompletedByValue'
+import SessionCompletedByValue from '../../../../server/enums/sessionCompletedByValue'
```

Comment:
It looks like quite a big PR, but basically its a rename of `ReviewPlanCompletedByValue` to `SessionPlanCompletedByValue`, and apart from the renamed file itself (👇 ) all the changes in this PR are change to the imports and references 👍